src/pages/Inventory.page.js

```diff
@@ -32,4 +35,31 @@ export class InventoryPage extends BaseSwagLabPage {
     async sortProductPriceHiLo() {
         await this.sortProductButton.selectOption('hilo');
     }
+
+    async addRandomProductsToCart() {
+        const allProductsCount = await this.inventoryItems.count();
+        const randomInt = randomIntFromInterval(1, allProductsCount);
+
+        this.randomInt = randomInt;
```

Comment:
Can we operate elements (get random elements) as we discussed on the Q&A session on Monday?

Reply:
Let's use this method https://github.com/letztersommer/playwright_tests_with_po/pull/2/files#diff-c079cd9d82ef3410cf2ca5a6ba5597e827fcc378aa7bc976270efe89a82f2908R1